Source/WebCore/platform/graphics/MediaSourcePrivate.cpp

```diff
@@ -192,7 +192,8 @@ void MediaSourcePrivate::updateTracksType()
     TracksType tracksCombinedTypes;
     for (auto type : m_tracksTypes.values())
         tracksCombinedTypes |= type;
-    m_tracksCombinedTypes = tracksCombinedTypes;
+    if (m_tracksCombinedTypes.exchange(tracksCombinedTypes) != tracksCombinedTypes) {
+    }
```

Comment:
What's the point of this `if` with an empty block?

Reply:
It’s a rebase error of another PR I just landed.